cmd/message_input.go

```diff
@@ -17,11 +17,13 @@ import (
 
 type MessageInput struct {
 	*tview.TextArea
+	replyMessageID int
```

Comment:
```suggestion
	replyMessageIdx int
```
`*ID` refers to a unique identifier for a resource, it is ambiguous what `replyMessageID` is supposed to be. Rename it to something like `replyMessageIndex` or `replyMessageIdx` for clarity.